src/main/java/seedu/address/model/role/DesiredCharacteristics.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.role;
+
+import java.util.Arrays;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the desired characteristics of a property that will interest a particular buyer.
+ * Individual characteristics are separated by semicolons.
+ */
+public class DesiredCharacteristics {
+    public static final String MESSAGE_CONSTRAINTS = "Desired characteristics can take any value, " +
+            "and can also be left blank.";
+
+    /*
+     * The first character of the desired characteristics must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input (when it should be a null wrapped in an Optional).
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
+
+    public static final String CHARACTERISTIC_DELIMITER = ";";
```

Comment:
I feel like comma as a delimiter would be more intuitive

Reply:
What if individual characteristics are punctuated with commas?

Reply:
ah i see, i would argue at that point there should be separate characteristics. but okays lets just keep our delimiters consistent